assets/performanceprofile/scripts/clear-irqbalance-banned-cpus.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# const
+SED="/bin/sed"
+# tunable - overridable for testing purposes
+IRQBALANCE_CONF="${1:-/etc/sysconfig/irqbalance}"
+
+[ ! -f ${IRQBALANCE_CONF} ] && exit 0
+
+${SED} -i '/^IRQBALANCE_BANNED_CPUS=/d' ${IRQBALANCE_CONF} || exit 0
```

Comment:
It probably is not really needed, but wouldn't the following be a bit more robust?
`'/^\s*IRQBALANCE_BANNED_CPUS\b/d'`